lib/utilities/default-targets.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
```

Comment:
This is totally outside the scope but here we go:
This matrix is the actual matrix of browsers in which ember itself is tedted. I believe that probably safari 9 should be added. Arguably today iOS9 is far more relevant than ie9 & 10 combined.